shared/editor/components/Styles.ts

```diff
@@ -124,6 +124,17 @@ font-size: 1em;
 line-height: 1.6em;
 width: 100%;
 
+.mention {
+  background: ${props.theme.listItemHoverBackground};
```

Comment:
Lets add a new item to the theme for this, `mentionBackground` – it can be the same color as `listItemHoverBackground` for now.

Reply:
Enhancement tip: Color mention nodes differently, each color representing a mentioned entity. Not a blocker, can be pursued later.